pkg/fixtures/image/reader.go

```diff
@@ -0,0 +1,51 @@
+package image
+
+import (
+	"embed"
+	"path/filepath"
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/jsonutil"
+	"github.com/stackrox/rox/pkg/utils"
+)
+
+const (
+	dir = "files"
+)
+
+var (
+	//go:embed files/*.json
+	fs embed.FS
+)
+
+// GetTestImages returns a slice of images for testing purposes. These images contain a snapshot of image scan.
+func GetTestImages(_ *testing.T) ([]*storage.Image, error) {
```

Comment:
In addition to this, would a func that can take a file name and return a single image be useful?

Reply:
It doesn't hurt to have it. I will club it with other changes in one of the follow-ups.